HOWTO_meta_quest.md

```diff
@@ -352,46 +356,47 @@ properly capture the necessary content.
 ### 2. Change the Debug app to the GFXRecon Quest replay app
 
 If you haven't already done so, first follow the instructions to enable
-the debug layer in section
-[3. Enable GFXReconstruct](#3-enable-gfxreconstruct)
-in the Capturing IGL Content section above.
+the debug layer in the
+[Enable GFXReconstruct Vulkan Capture](#5-enable-gfxreconstruct-vulkan-capture)
+section above.
 
-One change is to make sure that we:
-* Capture the correct application (here it is the replay application)
+One change is to make sure that:
+* The correct application name is used (here it is the Quest replay
+  application)
 * Adjust the capture file to use a different name than
-the one being replayed (or you will overwrite what you're reading)
+  the one being replayed (or it will overwrite what is being read)
```

Comment:
is the leading space what we want?

Reply:
It is to indicate it is part of the bullet above.  It works without the space, but is clearer in my opinion.